src/commands/init.js

```diff
@@ -1,57 +1,39 @@
 const fs = require('fs')
 const path = require('path')
 const jsYaml = require('js-yaml')
+const cache = require('../cache')
+const storage = require('../lib/storage')
 
 const supermodelConfig = require('../lib/supermodelConfig')
 
-// This function is duplicated in superlib
-// TODO: Establish URI / id manipulation library in superlib and remove it from here
-const { URL } = require('url')
-function isURL(subject) {
-  try {
-    new URL(subject)
-    return true
-  }
-  catch (e) {
-    return false
-  }
-}
-
-// Runs init command, creating a supermodel directory and initializing it with config
-//
-// @param {string} baseId - Base identifier of the supermodel package, 
-//   if the user is NOT logged in the identifier must be an absolute URI, if the user is logged the URI must be relative to her/his config 
-function runInit(baseId) {
-  // Verify the id
-  if (!isURL(baseId)) {
-    console.error(`Error: The baseId '${baseId}' must be a valid URL`)
-    process.exit(1)
-  } 
-
-  // Supermodel directory
-  const supermodelDir = path.join(process.cwd(), supermodelConfig.SUPERMODEL_DIR_NAME)
-  if (fs.existsSync(supermodelDir)) {
-    console.error('Error: Supermodel directory already exists')
+/**
+ * Runs init command, creating a supermodel directory and initializing it with config
+ *
+ * @param {string} layerPath Optional path to the layer to be created
+ */
+function runInit(layerPath) {
+  // Verify the user is logged in
```

Comment:
Toto bych dal do samostatneho modulu, protože to budu potřebovat hned v zápětí také volat při pull a push.